src/main/java/com/exasol/release/robot/AbstractPlatform.java

```diff
@@ -1,4 +1,4 @@
-package com.exasol;
+package com.exasol.release.robot;
```

Comment:
If I remember correct @redcatbear said we should not use `.` in packages as a separator, if it is not a real package. So you would have to sue `releaserobot` instead. That is not so readable but has the correct semantic. But maybe you should ask him...